novelwriter/dialogs/projectsettings.py

```diff
@@ -353,7 +354,8 @@ def __init__(self, parent: QWidget) -> None:
         self.sessGoalAuto.setChecked(data.sessGoalAuto)
         self.addRow(
             self.tr("Auto populate session goal?"), self.sessGoalAuto,
-            self.tr("Calculate Session goal based on target date assuming daily sessions. Overrides configured session goal below.")
+            self.tr("Calculate Session goal based on target date assuming daily sessions. \
+                    Overrides configured session goal below.")
```

Comment:
Please don't use backslash for line continuation. This is the correct way:
```suggestion
            self.tr(
                "Calculate Session goal based on target date assuming daily sessions. "
                "Overrides configured session goal below."
            )
```